DayPicker.js

```diff
@@ -2,14 +2,16 @@
 /* eslint-env node */
 
 var DayPicker = require('./lib/src/DayPicker');
+var DayPickerInput = require('./lib/src/DayPickerInput');
```

Comment:
Please don't change the shape of the module export like this -- it's a breaking change. If you want to import `DayPickerInput` you'll need to do it as `import { DayPickerInput } from "react-day-picker/DayPickerInput";` and define the types like this:

```
// index.d.ts stays the same
```

```
// DayPickerInput.d.ts
export class DayPickerInput ... {
    ...
}
```

I do think moving the `types/` folder contents to the root will be necessary for this, but you could make that happen only in the distribution if @gpbl doesn't want those at the root of the source repo.